benchmarks/dbscan/dbscan.cpp

```diff
@@ -0,0 +1,178 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "dbscan.hpp"
+
+#include <ArborX_DBSCAN.hpp>
```

Comment:
```suggestion
```

Reply:
No, I have it because I use `DBSCAN::Implementation` when dealing with command line argument `--impl` and I don't want to rely on transitivity.

Reply:
And I think that is wrong

Reply:
If you look at the changes, I removed that usage of `Implementation` in the parameters struct. Instead of relying on boost to parse that parameter, we now do it ourselves.